cms/static/sass/partials/_variables.scss

```diff
@@ -41,11 +41,16 @@ $transparent: rgba(0,0,0,0); // used when color value is needed for UI width/tra
 
 // +Colors - UXPL new pattern library colors
 // ====================
-$uxpl-blue-base: rgba(0, 116, 180, 1); // wcag2a compliant
-$uxpl-blue-hover-active: lighten($uxpl-blue-base, 7%); // wcag2a compliant
+$uxpl-blue-base: rgb(0, 117, 180); // wcag2a compliant
```

Comment:
You know - I would __LOVE__ that. I thought that Alisan was working on something, but it'd be a while? Happy to create a pattern here which she can use in her work, if you prefer. Or we can sync up later. Thoughts?

Sandbox is up-to-date!